vivisect/__init__.py

```diff
@@ -2138,14 +2182,29 @@ def isLocType(self, va, ltype):
             return False
         return tup[L_LTYPE] == ltype
 
-    def getLocation(self, va, range=False):
+    def getLocation(self, va, range=True):
         """
         Return the va,size,ltype,tinfo tuple for the given location.
         (specify range=True to potentially match a va that is inside
-        a location rather than the beginning of one)
-        """
-        # TODO: range=True does nothing
-        return self.locmap.getMapLookup(va)
+        a location rather than the beginning of one, this behavior
+        only affects strings/substring retrieval currently)
+        """
+        loc = self.locmap.getMapLookup(va)
+        if not loc:
+            return loc
```

Comment:
qua!?  return None? or False?  or what are we expecting here?  do we want to just hand whatever failure through?